views/partials/header.hbs

```diff
@@ -0,0 +1,36 @@
+<header class="main-header">
+    <div class="wrapper">
+        <nav class="row">
+            <div class="col-xs-4 col-md-3 main-header__logo">
+                <a href="/"><img src="/static/images/logo_2x.png" alt="Sabka Bazar logo" class="logo"></a>
+            </div>
+            <div class="col-xs-8 col-md-9 main-header__navlist" id="header-nav">
+                <!-- Hamburger Menu -->
+                <label for="header-hamburger">&#9776;</label>
+                <input type="checkbox" id="header-hamburger"/>
+                <div class="main-header__navlinks">
+                    <ul class="main-header__main-nav left">
+                        <li><a href="/">{{menuHome}}</a></li>
+                        <li><a href="/product-listing">{{menuProduct}}</a></li>
+                    </ul>
+                    <ul class="main-header__main-nav right end-xs">
+                        <li><a href="/login">{{menuLogin}}</a></li>
+                        <li><a href="/register">{{menuRegister}}</a></li>
+                    </ul>
+                </div>
+                <span>
+                    <a href="/cart" class="main-header__cart">
```

Comment:
can you try passing all the routes through api data, make it dynamic. That way if you plan to navigate so some other route, you can control it via api itself and not have to update code here. Do that in all files